### DISCUSSION TITLE
Model Export plugin: Export translucency

### DISCUSSION THREAD

**Discussion by Apteryxian** [runelite/runelite#18420] [Ideas]
It would be awesome if the exported .mtl files had translucency for items such as ghostly robes and the abyssal tentacle.

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.
